roles/setup_barman/templates/barman.rsync.template

```diff
@@ -0,0 +1,11 @@
+[{{ ansible_hostname }}]
```

Comment:
Will ansible hostname include the hostname of the target node?

Reply:
Yes, in this case, the target node is the current host.